src/main/flight/mixer.c

```diff
@@ -511,6 +513,13 @@ static float calcEzLandLimit(float maxDeflection)
         ezLandLimit = maxDeflection / mixerRuntime.ezLandingThreshold; // normalised 0 - 1
         ezLandLimit = fmaxf(ezLandLimit, mixerRuntime.ezLandingLimit); // stay above the minimum
     }
+    // calculate limit to where the mixer can raise the throttle based on RPY stick deflection
+    // TODO sanity checks like number of sats, dop, accuracy?
+    if (speed < mixerRuntime.ezLandingSpeed && mixerRuntime.ezLandingSpeed > 0.0f) {
+        const float speed_limit = speed / mixerRuntime.ezLandingSpeed;  // normalised 0 - 1
+        DEBUG_SET(DEBUG_EZLANDING, 5, lrintf(speed_limit * 10000.0f));
+        ezLandLimit = fmaxf(ezLandLimit, speed_limit);
+    }
```

Comment:
```suggestion
if (mixerRuntime.ezLandingSpeed > 0.0f) {
    if (speed < mixerRuntime.ezLandingSpeed && ) {
        const float speed_limit = speed / mixerRuntime.ezLandingSpeed;  // normalised 0 - 1
        DEBUG_SET(DEBUG_EZLANDING, 5, lrintf(speed_limit * 10000.0f));
        ezLandLimit = fmaxf(ezLandLimit, speed_limit);
    } else {
        ezLandLimit = 1.0f;
    }
}
```

But maybe `fmaxf` logic is wrong ..

Reply:
I'm pretty sure you're right, the logic is wrong if speed > ezLandingSpeed. 
Writing a fix now.

Reply:
Should be fixed now